openedx/features/edly/message_types.py

```diff
@@ -0,0 +1,25 @@
+from openedx.core.djangoapps.ace_common.message import BaseMessageType
+
+
+class OutlineChangesNotification(BaseMessageType):
+    """
+    A message for notifying user about new changes in course.
+
+    """
+    APP_LABEL = 'edly'
+
+    def __init__(self, *args, **kwargs):
+        super(OutlineChangesNotification, self).__init__(*args, **kwargs)
+        self.options['transactional'] = True
```

Comment:
We should add some documentation here about why are we enabling this flag.

Reply:
That's not required in our case and it is removed in the latest push.